home/views.py

```diff
@@ -0,0 +1,16 @@
+from django.http import HttpResponseRedirect
+from wagtail.admin.views.pages.listing import index as wagtailadmin_explore
+
+from home.models import FooterIndexPage
+
+
+def override_wagtailadmin_explore_default_ordering(request, parent_page_id):
```

Comment:
Not a big fan of this solution. I think there may be a better approach by using the [construct-explorer-page-queryset wagtail hook][1].

I would also suggest that perhaps it's not such a big deal for manual ordering (ordering=ord) to be enabled by default, but it is important for the ordering to be by 'path' regardless.

[1]: https://docs.wagtail.io/en/stable/reference/hooks.html#construct-explorer-page-queryset

Reply:
yes, sure if enabling ordering by default is not a big then it far better to use wagtail hook. Changes addressed.

Reply:
also, should we do this just for footer pages or for all the pages?

Reply:
Perhaps it should be for other pages too, but just footer pages is fine for now.